.changeset/violet-impalas-grin.md

```diff
@@ -0,0 +1,7 @@
+---
+'@strapi/icons': major
```

Comment:
Could you make it a minor instead?

(I'm not sure it actually matters since the tool is in pre-release mode, but I don't want to take any chances)

Reply:
Sure on it.

Reply:
@remidej changed it from major to minor.